### DISCUSSION TITLE
Check the logs of a user

### DISCUSSION THREAD

**Discussion by asoMan1369** [vpnhood/VpnHood#290] [General]
How do i can check and read logs of a user in ubuntu?

**Comment by trudyhood** [comment]
check /opt/VpnHoodServer/storage/logs/
sessions.log
track.log

In session.log, you can find the user token that establishes the session. You can detect copyright infringement or attack abuse by these files. However, due to respecting user privacy, VpnHood redacts User Requested destination IP addresses, and there is no option to stop it. You need to download and recompile the server source code if you want to disable destination IP address redacting.
